docs/usage/notes.rst

```diff
@@ -278,10 +281,19 @@ won't free disk space, they merely tag data as deleted in a new transaction.
 Be aware that as soon as you write to the repo in non-append-only mode (e.g. prune,
 delete or create archives from an admin machine), it will remove the deleted objects
 permanently (including the ones that were already marked as deleted, but not removed,
-in append-only mode).
+in append-only mode). Automated edits to the repository (such as a cron job running
+``borg prune``) will render append-only mode moot if data is deleted.
```

Comment:
not sure about what new information this is telling me?

Reply:
edits to the repository -> write operations to the repository?
if data is deleted -> if data was maliciously/accidentally deleted?

is that what you meant?

Reply:
That's what I meant. I saw several issues where people were confused about aspects of append-only mode mentioned in the docs. I thought spelling out "a cron job that writes to the repository will make append-only mode useless" would make it more clear.

Reply:
ok.